SentinelOne/cloud_funnel2.0/_meta/manifest.yml

```
@@ -0,0 +1,10 @@
+uuid: 8c1bee36-d516-42f9-9b6f-a8e4dcac3d1d
```

Please could you change this uuid? (duplicated with the one of Deep-Visibility format)
```suggestion
uuid: 40deb162-6bb1-4635-9c99-5c2de7e1d340
```